OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameMenuLogic.cs

```diff
@@ -145,7 +145,11 @@ public IngameMenuLogic(Widget widget, World world, Action onExit, WorldRenderer
 			};
 			var surrenderButton = menu.Get<ButtonWidget>("SURRENDER");
 			surrenderButton.IsVisible = () => world.Type == WorldType.Regular;
-			surrenderButton.IsDisabled = () => (world.LocalPlayer == null || world.LocalPlayer.WinState != WinState.Undefined) || hasError;
+			surrenderButton.IsDisabled = () =>
+			{
+				var surrenderVisCheck = world.LocalPlayer == null || world.LocalPlayer.WinState != WinState.Undefined || world.Map.Visibility == MapVisibility.MissionSelector;
```

Comment:
Ah right - that makes sense.

This might need to be `world.Map.Visibility.HasFlag(MapVisibility.MissionSelector)` - most of the other checks do it that way.